packages/js/manifests/wrap/src/formats/wrap.info/compare.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  MethodDefinition, PropertyDefinition,
+} from ".";
+
+export function compareAbis(
```

Comment:
Since compareAbis doesn't explicitly depend on the manifest version, could it be a non-generated (normal) method?

Reply:
I think it should be dependant on `Abi` version - Wdyt?

Reply:
Sure, I just meant to say that right now it's not explicitly dependent on an `Abi` version.

It looks like the mustache template doesn't have a way to adapt to changes in the schema. The mustache template is static--it doesn't depend on any outside configuration, such as the `Abi`. So if the `Abi` structure changes, someone will have to write a new mustache template with a new method. Whether it's generated or not, it's going to break just the same.

I might be missing something, but it looks that way to me at the moment.

What are the advantages of generating this method versus not? Can we make it robust to changes in `Abi` structure, so that it updates to accommodate changes in the `Abi`?

Reply:
I will move this to `js/wrap/src`, I also changed the name of it to `compareSignatures`, I found it more explicit - lmk your thoughts!

Reply:
LGTM!